src/plugin.ts

```diff
@@ -61,6 +62,19 @@ export function fastifyApollo<
 	return fp(async (fastify, options) => {
 		const { path = "/graphql", method = ["GET", "POST", "OPTIONS"], ...handlerOptions } = options;
 
+		// Fastify has a default JSON parser that throws a specific error when the body is empty.
+		// A GraphQL server which is spec compliant (per the graphql-over-http spec) should respond
+		// with a JSON object with an `errors` key when the body is empty.
+		fastify.removeContentTypeParser(["application/json"]);
+		// @ts-ignore
+		// TODO: Something with all the Fastify generics is messing up the types with the `fastify` object
```

Comment:
@olyop note this TODO...if you're familiar with the generics maybe you know what I'm doing wrong here?